packages/react-components/src/organisms/GroupNoEvents.tsx

```diff
@@ -1,38 +1,23 @@
-import { css } from '@emotion/react';
-import { getButtonStyles } from '../button';
 import { crossInCircleIcon } from '../icons';
-import { perRem } from '../pixels';
-
-const titleStyles = css({
-  padding: `20px`,
-
-  fontSize: `${26 / perRem}em`,
-});
-
-const exploreEventsStyles = css(getButtonStyles({ primary: true }), {
-  textDecoration: 'none',
-});
+import NoEventsLayout from './NoEventsLayout';
 
 const GroupNoEvents: React.FC<{ past?: boolean; link: string }> = ({
   past,
   link,
 }) => (
-  <main css={{ textAlign: 'center' }}>
+  <NoEventsLayout>
     {crossInCircleIcon}
-    <div css={titleStyles}>
-      <strong>
-        This group doesn’t have any {past ? ' past ' : ' upcoming '} events!
-      </strong>
-    </div>
-    <span>
+    <NoEventsLayout.Title>
+      This group doesn't have any {past ? ' past ' : ' upcoming '} events!
```

Comment:
I would much prefer this to be a single component that took in these strings as props.